### PR TITLE
Don't set test check when tess are not configured

### PR DESCRIPTION
**Pull request by dhodovsk**
Fixes https://github.com/packit-service/packit-service/issues/272

### CODE REVIEW

File: packit_service/worker/copr_build.py [hunk 1]
```diff
@@ -96,6 +96,16 @@ def get_job_copr_build_metadata(self) -> Optional[JobConfig]:
...
 
+    def cfg_has_job_tests(self) -> bool:
+        """
+        Check if there is tests job configured
+        :return: bool
+        """
+        for job in self.package_config.jobs:
+            if job.job == JobType.tests:
+                return True
+        return False
```
[the remark this answers is not in the document]
Comment (dhodovsk) [reply packit_service/worker/copr_build.py]: its the same as [get_job_copr_build_metadata](https://github.com/packit-service/packit-service/blob/02336934fcc5394d8498cdc0402487a43546410f/packit_service/worker/copr_build.py#L89)

[the remark this answers is not in the document]
Comment (jpopelka) [reply packit_service/worker/copr_build.py]: no, it's not. `get_job_copr_build_metadata()` returns `Optional[JobConfig]` while your method returns `bool`

[the remark this answers is not in the document]
Comment (dhodovsk) [reply packit_service/worker/copr_build.py]: right

File: Dockerfile [hunk 2]
```diff
@@ -15,6 +15,7 @@ RUN cd /src/ \
     && ansible-playbook -vv -c local -i localhost, files/install-deps.yaml \
     && dnf clean all
 
+RUN pip3 install git+https://github.com/packit-service/packit-service
```
Comment (lachmanfrantisek) [inline Dockerfile]: Why is this needed? I would expect using local dir for that.

Comment (dhodovsk) [reply Dockerfile]: Oh, that was just a leftover from my local testing to quickly workaround [incompatibility](https://github.com/packit-service/packit/pull/614). Thanks for noticing.

Comment (lachmanfrantisek) [reply Dockerfile]: Thanks!
